src/main/java/com/rarchives/ripme/utils/RipUtils.java

```diff
@@ -9,97 +22,91 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import org.apache.commons.lang.math.NumberUtils;
-import org.apache.log4j.Logger;
-import org.jsoup.Jsoup;
-import org.jsoup.nodes.Document;
-import org.jsoup.nodes.Element;
+public class RipUtils {
 
-import com.rarchives.ripme.ripper.AbstractRipper;
-import com.rarchives.ripme.ripper.rippers.ImgurRipper;
-import com.rarchives.ripme.ripper.rippers.ImgurRipper.ImgurAlbum;
-import com.rarchives.ripme.ripper.rippers.ImgurRipper.ImgurImage;
-import com.rarchives.ripme.ripper.rippers.VidbleRipper;
-import com.rarchives.ripme.ripper.rippers.video.GfycatRipper;
-import com.rarchives.ripme.ripper.rippers.EroShareRipper;
+    private static final Logger LOGGER = Logger.getLogger(RipUtils.class);
+    private static final String IMGUR_COM = "imgur.com";
+    private static final String HTTP = "http://";
 
-public class RipUtils {
-    private static final Logger logger = Logger.getLogger(RipUtils.class);
+    private RipUtils() {
+    }
 
     public static List<URL> getFilesFromURL(URL url) {
         List<URL> result = new ArrayList<URL>();
 
-        logger.debug("Checking " + url);
+        LOGGER.debug("Checking " + url);
```

Comment:
Lowercase logger is used throughout ripme. Sure it's a convention for "constants" to be in all caps, but that static final variable isn't even benefitting from pretending to be a constant. Sure you can't reassign it, but as an output logger, it's state changes. I don't think anything useful is documented there. I'd prefer keeping it lowercase `logger` for consistency with the rest of the program and to reduce the size of unnecessary diffs. If you're serious about this change, it needs to change in the whole project.